test/integration/get-payload.test.ts

```diff
@@ -85,9 +85,20 @@ describe("getPayload", () => {
     const promise = getPayload(request);
 
     // we emit data, to ensure that the body attribute is preferred
-    request.emit("data", "bar");
+    request.emit("data", Buffer.from("bar"));
```

Comment:
It was working, because of 
https://github.com/octokit/webhooks.js/blob/9a9593a4325a0e4bc3e05409382c9bf3cd80ddd7/src/middleware/node/get-payload.ts#L38

So the string "bar" would be then "bar".toString()

I guess in the moment where we emit two times strings, it would because Buffer.concat expects only Uint8Array and Buffers.